test/coverage-test/test/configuration-options.test-d.ts

```diff
@@ -209,12 +205,6 @@ test('reporters, with options', () => {
 
   assertType<Coverage>({
     reporter: [
-      // @ts-expect-error -- teamcity report option on html reporter
-      ['html', { blockName: 'string' }],
-
-      // @ts-expect-error -- html-spa report option on json reporter
-      ['json', { metricsToShow: ['branches'] }],
-
```

Comment:
I wasn't able to keep options types supported after adding support for `string` as reporter name. Built-in reporters' options are still prompted by intellisense though.